src/api/telemetry/TelemetryAPI.js

```diff
@@ -166,25 +196,27 @@ export default class TelemetryAPI {
     }
 
     /**
-     * @private
-     * Though used in TelemetryCollection as well
+     * @param {TelemetryRequestOptions} options options for the telemetry request
+     * @returns {TelemetryRequestOptions} the options, with defaults filled in
      */
-    standardizeRequestOptions(options) {
-        if (!Object.prototype.hasOwnProperty.call(options, 'start')) {
+    standardizeRequestOptions(options = {}) {
+        if (!Object.hasOwn(options, 'start')) {
             options.start = this.openmct.time.bounds().start;
         }
 
-        if (!Object.prototype.hasOwnProperty.call(options, 'end')) {
+        if (!Object.hasOwn(options, 'end')) {
```

Comment:
Maybe start and end bounds should default to the bounds of the provided timeContext, if defined, instead of always defaulting to the global time context.